ontology-service/src/test/java/no/sikt/nva/OntologyControllerTest.java

```diff
@@ -36,11 +47,72 @@ public static void cleanupSpec() {
     }
 
     @Test
-    void shouldReturnOk() {
-        AwsProxyRequest request = new AwsProxyRequestBuilder("/file", HttpMethod.GET.toString())
-                .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON)
+    void shouldReturnTurtle() {
```

Comment:
shouldReturnTurtleWhenAcceptHeaderIsApplicationTurtle. In general add the "when" condition in all test names